web/pom.xml

```diff
@@ -21,7 +21,8 @@
         <java.version>1.7</java.version>
         <spring.version>4.3.8.RELEASE</spring.version>
         <spring.boot.version>1.5.3.RELEASE</spring.boot.version>
-        <oncotree.core.version>0.0.1-SNAPSHOT</oncotree.core.version>
+        <oncotree.core.version>1.0.0-SNAPSHOT</oncotree.core.version>
```

Comment:
This looks good, but I think the pom for core also needs to be updated.  Unit tests failed with "Could not find artifact org.mskcc.oncotree:core:jar:1.0.0-SNAPSHOT"

Reply:
@sheridancbio ah good catch. Just updated.